objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestNoProcessor.java

```diff
@@ -0,0 +1,502 @@
+/*
+ * Copyright (c) 2018 Vantiq, Inc.
+ *
+ * All rights reserved.
+ *
+ * SPDX: MIT
+ */
+
+package io.vantiq.extsrc.objectRecognition.neuralNet;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import io.vantiq.client.BaseResponseHandler;
+import io.vantiq.client.Vantiq;
+import io.vantiq.client.VantiqError;
+import io.vantiq.client.VantiqResponse;
+import io.vantiq.extsrc.objectRecognition.exception.ImageProcessingException;
+import okhttp3.Response;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class TestNoProcessor extends NeuralNetTestBase {
+
+    static final String OUTPUT_DIR = System.getProperty("buildDir") + "/resources/out";
+    static final int SAVE_RATE = 2; // Saves every other so that we can know it counts correctly
+    static final String NOT_FOUND_CODE = "io.vantiq.resource.not.found";
+    static final String CONFIG_JSON_LOCATION = "src/test/resources/";
+
+    static NoProcessor noProcessor;
+    static Vantiq vantiq;
+    static VantiqResponse vantiqResponse;
+
+    static List<String> vantiqSavedFiles = new ArrayList<>();
+
+    static final String timestampPattern = "\\d{4}-\\d{2}-\\d{2}--\\d{2}-\\d{2}-\\d{2}\\.jpg";
+    static final String sameTimestampPattern = "\\d{4}-\\d{2}-\\d{2}--\\d{2}-\\d{2}-\\d{2}\\(\\d{1}\\)\\.jpg";
```

Comment:
Shouldn't `\\(\\d{1}\\)` be `\\(\\d+\\)` or something like that (that is -- take any number in there, not just single digit ones)